test/scripts/git_clone.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# To run this script from your workspace execute this command
+#   ./test/scripts/run.sh /mnt/blob_mnt /mnt/blobfusetmp ./config.yaml ./v1.cfg 2&> results.txt
```

Comment:
"run.sh" commands in comments is not correct

Reply:
fixed, will push in a bit